src/main/java/com/contrastsecurity/http/TraceFilterForm.java

```diff
@@ -123,7 +127,10 @@ public String toQuery() throws UnsupportedEncodingException {
 
     if (expand != null && !expand.isEmpty()) {
       filters.add(
-          "expand=" + expand.stream().map(Object::toString).collect(Collectors.joining(",")));
```

Comment:
do you think we need a test in this method `toQuery()` to check if the filter string is built as expected?

Reply:
Added tests.